projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -42,6 +45,12 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   @Input('clrAccordionPanelDisabled') @HostBinding('class.clr-accordion-panel-disabled') disabled = false;
   @Input('clrAccordionPanelOpen') panelOpen = false;
 
+  @Input('clrAccordionPanelHeading') heading = false;
```

Comment:
```suggestion
  @Input('clrAccordionPanelHeadingEnabled') headingEnabled = false;
```

A property just named `heading` is confusing to read. This property does not hold a "heading". It holds a feature flag.

Reply:
Corrected in commit: cc43f46